spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -293,6 +303,13 @@ def main(argv: Sequence[str]):
     if arguments.l is not None:
         fname_landmarks = arguments.l
         label_type = 'body'
+    elif arguments.lrootlet is not None:
+        fname_rootlets = arguments.lrootlet
+        label_type = 'rootlet'
+        if arguments.ldisc is not None:
+            fname_landmarks = arguments.ldisc
+        else:
+            raise ValueError('Single disc label should be provided with rootlets labels.')
```

Comment:
Sorry if this was not clear, yes the flag `-ldisc`, but I ma using it for rootlets midpoint as a bypass, but it may be confusing for the user. I am maybe thinking of computing the rootlets midpoints inside `sct_register_to_template` insetead, and the user would only need to provide `-lrootlet` does this seam okay?

Reply:
Yes, that makes perfect sense to me!

Though, I am curious, because for vertebral labeling + registration, [we extract the midpoint](https://spinalcordtoolbox.com/stable/user_section/tutorials/vertebral-labeling/extracting-specific-labels-for-registration.html) and [use the midpoints for the `-l` argument](https://spinalcordtoolbox.com/stable/user_section/tutorials/registration-to-template/template-registration/applying-the-registration-algorithm.html).

So, to make sure I understand: Rootlets-based registration requires rootlets seg + rootlets midpoint for all rootlets (hence automatic extraction), while vertebral level-based registration requires just the midpoints of 2 (or more) specific levels (hence using `sct_label_utils` to pick specific levels).

Reply:
> So, to make sure I understand: Rootlets-based registration requires rootlets seg + rootlets midpoint for all rootlets (which is why automatic extraction would be beneficial), while vertebral level-based registration requires just the midpoints of 2 (or more) specific levels (hence using `sct_label_utils` to pick specific levels).

yes exactly!